README.md

```diff
@@ -1,62 +1,28 @@
 # genval [![Build Status](https://travis-ci.org/gojuno/genval.svg?branch=master)](https://travis-ci.org/gojuno/genval)
 
-Generates Validate() methods for all structs in pkg by tags
+Generates Validate() methods for all structs in package by tags
 - no reflection in generated code - it means fast  
-- generator not needed on runtime
-- possibilities to override generated behavior for local purposes
+- possibilities to override generated behavior
 - can be used as `//go:generate genval pkg` 
```

Comment:
@mkorolyov what do you think about such behavior:
we can use either
```genval directory```
or
```genval -d directory -p package```

If we pass just single argument then it will used as both flags `-d` & `-p`. I think it`s more comfortable to use because situation when directory names differs from package name is very rare.